python/packages/nisar/products/insar/utils.py

```diff
@@ -573,3 +573,45 @@ def generate_insar_subswath_mask(ref_rslc_obj,
         (len(azi_idx_arr),
          len(rg_idx_arr))).astype(np.uint8)
 
+def get_static_layers_data_access(static_layers_data_access_runconfig,
```

Comment:
Once #100 is merged, please simply call that function. No need for duplicate code. :)